### ISSUE TITLE
cuda.h not found

### ISSUE THREAD

**Issue by ScarWar** [iot-salzburg/gpu-jupyter#86]
Hello,
I think we are missing an env. variable in the Dockerfile on the location of the cuda headers. 

I ran the docker on my university GPU cluster using [`udocker` ](https://indigo-dc.github.io/udocker/), and I'm using a library called [`KeOps`](https://www.kernel-operations.io/keops/index.html) which uses cuda intensively. 
But every time I tried to import `keops` in my project I had a problem with `cuda.h` file not being found, although when I ran `torch.cuda.is_avaliable()` I was getting `True`.
I was searching for a solution for a long time, and after looking at the keops-full [dockerfile](https://github.com/getkeops/keops/blob/main/Dockerfile) I found a tiny but essential env. variable `CUDA_PATH`, which is set to `/opt/conda/`, which indeed contains `cuda.h` file.

After setting in `~/.bashrc` the env. variable `export CUDA_PATH=/opt/conda/` and restarting the docker I ran the import again and found success.

```bash
(base) root@ise-pheno-01:~# python -c "from pykeops.torch import LazyTensor""
[KeOps] Compiling cuda jit compiler engine ... OK
[pyKeOps] Compiling nvrtc binder for python ... OK
```

I think that this env. is missing and must be added to the dockerfile

**Comment by mathematicalmichael** [comment]
oh interesting. Is that env variable specific to your library? If so I'd argue that your use case requires modifying the dockerfile to install your binaries + set up the env variables they need.

If it's something required by tflow/torch/other-libs, then I can see an argument for including it by default.

**Comment by ScarWar** [comment]
Thanks for the quick repaly

No, the env. var is not specific to any library, but rather it is related to the CUDA SDK location. AFAIK, tf and torch come with their own `cuda.h`, but many other libraries, including Nvidia's, need it set correctly ([quick search results](https://cs.github.com/?scopeName=All+repos&scope=&q=CUDA_PATH)).

**Comment by mathematicalmichael** [comment]
Ah I see. Well then I think we should add this!

Seems like a simple addition to the Dockerfile. Would you like to open a PR or should I do so on your behalf?

**Comment by ChristophSchranz** [comment]
Hi @ScarWar 
this is definitely important and will be part of the next release.
Thank you!

**Comment by ChristophSchranz** [comment]
Solved with PR #87
